### PR TITLE
Implement IDREF element reflection

### PR DESCRIPTION
**Pull request by mrego**
#### dcb5be262b0642e25ffd6226061d6cf9b18cf5c9
<pre>
Implement IDREF element reflection
<a href="https://bugs.webkit.org/show_bug.cgi?id=196843">https://bugs.webkit.org/show_bug.cgi?id=196843</a>

Reviewed by Ryosuke Niwa.

Enable AriaReflectionForElementReferencesEnabled experimental flag by
default.

* Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml:

Canonical link: <a href="https://commits.webkit.org/252590@main">https://commits.webkit.org/252590@main</a>
</pre>

### CODE REVIEW

**Comment by rniwa** [comment]
Do we pass all the tests?

**Comment by mrego** [comment]
> Do we pass all the tests?

Yes we pass all WPT tests.

As commented on the bug, the HTML spec PR has been merged and the ARIA PR has been approved too.

**Comment by rniwa** [comment]
Is it possible to wait until the end of July to land this?

**Comment by mrego** [comment]
> Is it possible to wait until the end of July to land this?

Yes, sure. Just let me know when you think it might be a good moment to enable this.

**Comment by rniwa** [comment]
Yeah, will do. Thank you!

**Comment by cdumez** [comment]
FYI, it is not a good time to enable the feature on trunk. However, there seem to be some failures on Mac EWS.

**Comment by rniwa** [comment]
We should definitely fix https://github.com/WebKit/WebKit/pull/2486 before enabling this.

**Comment by mrego** [comment]
> FYI, it is not a good time to enable the feature on trunk. However, there seem to be some failures on Mac EWS.

@cdumez did you mean "it is now a good time"?

The Mac EWS failures are gone after @rniwa's patch.

> We should definitely fix https://github.com/WebKit/WebKit/pull/2486 before enabling this.

Thanks for fixing this.

**Comment by cdumez** [comment]
> > FYI, it is not a good time to enable the feature on trunk. However, there seem to be some failures on Mac EWS.
> 
> @cdumez did you mean "it is now a good time"?

Yes I did! What a confusing typo, sorry.

**Review by rniwa — APPROVED** [review]
r=me

**Comment by rniwa** [comment]
Thanks for waiting. We can enable it now.

**Comment by rniwa** [comment]
Actually,  that may have been premature. As far as I can tell, this isn't hooked up with the actual accessibility object tree at all.

**Pull request merged**
